ros2_batch_job/__main__.py

```diff
@@ -162,6 +162,19 @@ def main(sysargv=None):
 def get_args(sysargv=None):
     parser = argparse.ArgumentParser(
         description="Builds the ROS 2 repositories as a single batch job")
+
+    # TODO(clalancette): Python 3.6 on RHEL-8 doesn't have the 'extend' action
+    # available in argparse.  If we are on a Python version earlier than 3.8
+    # (where 'extend' was added), register it ourselves.
+    if sys.version_info.major == 3 and sys.version_info.minor < 8:
```

Comment:
```suggestion
    if sys.version_info < (3, 9):
```

Reply:
Good call, thanks.  I'll just re-run the very short CI here to ensure that it is good.